gallery/src/styles/NxAlert/NxAlertErrorExample.tsx

```diff
@@ -9,8 +9,8 @@ import { NxFontAwesomeIcon, NxCloseButton } from '@sonatype/react-shared-compone
 import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
 
 const NxAlertErrorExample = () =>
-  <div className="nx-alert nx-alert--error">
-    <NxFontAwesomeIcon icon={faExclamationCircle}/>
+  <div className="nx-alert nx-alert--error" role="alert" aria-atomic={true}>
```

Comment:
`role="alert"` implies `aria-atomic`.  The component just sets `aria-atomic` universally since some alerts don't have a role.  But in some cases it is redundant.

Reply:
what about role="status"? does that imply it as well?

Reply:
^^ it does, so i'll remove it from there as well.